code-challenges.rb

```diff
@@ -21,6 +34,9 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+album1.delete "aeiou"
+album2.delete "aeiou"
+album3.delete "aeiou"
```

Comment:
This will get the job done, but the goal is to create a single piece of reusable code. If you wrap this functionality into a method you can pass any given string and get the result.